pom.xml

```diff
@@ -30,11 +64,53 @@ limitations under the License.
     <maven>3.1.0</maven>
   </prerequisites>
 
```

Comment:
You might wish to have the following at the end of your `pom.xml`:
```
  <distributionManagement>
    <snapshotRepository>
      <id>ossrh</id>
      <url>https://oss.sonatype.org/content/repositories/google-snapshots/</url>
    </snapshotRepository>
    <repository>
      <id>ossrh</id>
      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
    </repository>
  </distributionManagement>
```
It's not required, but it will make it easier to test things.